### PR TITLE
Story/765 jwt token authentication

### PR DESCRIPTION
**Pull request by johevemi**
## Description

This PR adds JWT token authentication

## References

* [#765](https://app.zenhub.com/workspaces/joules-5fd22a1a59e4740018f5ab5a/issues/energinet-datahub/geh-aggregations/765)

### CODE REVIEW

File: source/dotnet/IntegrationTests.Core/Authorization/AuthorizationConfiguration.cs [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Energinet.DataHub.Core.FunctionApp.TestCommon.Configuration;
+using Microsoft.Extensions.Configuration;
+
+namespace Energinet.DataHub.Wholesale.IntegrationTests.Core.Authorization;
+
+public class AuthorizationConfiguration
```
Comment (BjarkeMeier) [inline source/dotnet/IntegrationTests.Core/Authorization/AuthorizationConfiguration.cs:20]: Looks like a candidate to be included in TestCommon. How about moving to the `TestCommon` folder?

Comment (johevemi) [reply source/dotnet/IntegrationTests.Core/Authorization/AuthorizationConfiguration.cs:20]: Agree this could be shared

**Review by BjarkeMeier — COMMENTED** [review]
What happens if I clone the repo and execute the test suite? Will I see failing integration tests?

**Comment by BjarkeMeier** [comment]
> If you don't have the integrationtest.local.setting.json file it will fail, but other than that I don't see it failing

Shouldn't we provide a better DX - especially as this is open-source and we want to encourage external contribution?

**Comment by johevemi** [comment]
> > If you don't have the integrationtest.local.setting.json file it will fail, but other than that I don't see it failing
> 
> Shouldn't we provide a better DX - especially as this is open-source and we want to encourage external contribution?

I totally agree, but I am not sure that it is possible to change atm because of how TestCommon works. I think we should bring this issue to our DevOps team so we can get a better understanding of the issue

**Review by BjarkeMeier — APPROVED** [review]

**Pull request merged**